docs/_basic/listening_modals.md

```diff
@@ -1,21 +1,23 @@
 ---
-title: Listening for view submissions
+title: Listening for view changes
```

Comment:
```suggestion
title: Listening to views
```
This mirrors our **Listening to actions** section

Reply:
Yes, "view changes" can be confusing. I agree with this suggestion.